go/vt/vttablet/tabletserver/gc/tablegc.go

```diff
@@ -251,7 +257,11 @@ func (collector *TableGC) operate(ctx context.Context) {
 		case <-tableCheckTicker.C:
 			{
 				log.Info("TableGC: tableCheckTicker")
-				_ = collector.checkTables(ctx, dropTablesChan, transitionRequestsChan)
+				gcTables, err := collector.readTables(ctx)
+				if err != nil {
+					log.Errorf("TableGC: error while reading tables: %+v", err)
+				}
```

Comment:
Should we `break` here to avoid running line `264`?

Reply:
Fixed!